README.md

```diff
@@ -39,7 +39,12 @@ Usage: chart_test.sh <options>
     --verbose         Display verbose output
     --no-lint         Skip chart linting
     --no-install      Skip chart installation
+    --chart-all       Lint/install all charts
+    --chart           Lint/install:
```

Comment:
I'd go for `--all-charts` or maybe just `--all` and `--charts` instad of `--chart`.